move-to-sdk/test/io/sphere/sdk/facets/MultiSelectFacetTest.java

```diff
@@ -0,0 +1,42 @@
+package io.sphere.sdk.facets;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MultiSelectFacetTest {
+    private static final String KEY = "multi-select-facet";
+    private static final String LABEL = "Select options";
+    private static final List<FacetOption<String>> OPTIONS = asList(
+            FacetOption.of("one", 30, true),
+            FacetOption.of("two", 20, true),
+            FacetOption.of("three", 10, false));
+
+    @Test
+    public void createsInstance() throws Exception {
+        final MultiSelectFacet<String> facet = MultiSelectFacetBuilder.of(KEY, LABEL, OPTIONS, true)
+                .setThreshold(3L)
```

Comment:
I used the IDE generator the third time I created the builders and I forgot to change it (: